fairscale/optim/oss.py

```diff
@@ -640,9 +628,11 @@ def _setup_bucket_strategy(self) -> None:
                         # This parameter becomes a view of the bucket
                         offset_next = offset + param.numel()
 
-                        self.buckets[device][dst_rank][offset:offset_next].copy_(param.data.flatten())
-                        param.data = self.buckets[device][dst_rank][offset:offset_next].view_as(param.data)
-
+                        try:
```

Comment:
I assume random try/catch is for debugging purposes since this is a draft :P when does this crash? any ideas?

Reply:
yes that's a mistake, sorry about that ! closing that today hopefully, different states to handle with pytorch 1.5 and cleanup needed